tests/acceptance/stepDefinitions/filesContext.js

```diff
@@ -1530,3 +1534,75 @@ function assertIncludesMessage(messageArr, message) {
     return false
   }
 }
+
+Then('the decline action button should be visible on the shared-with-me page', async function () {
+  const declineBatchActionButtonVisibility =
+    await sharedWithMePage.isDeclineBatchButtonVisibleForAutoAccept()
+  assert.ok(
+    declineBatchActionButtonVisibility,
+    `Expected decline action button to be visible but was not`
+  )
+})
+
+Then(
+  'the delete action button should be not visible on the shared-with-others page',
+  async function () {
+    let deleteBatchActionButtonVisibility =
+      await sharedWithOthersPage.isDeleteBatchButtonVisibleForAutoAccept()
+    if (deleteBatchActionButtonVisibility === -1) {
+      deleteBatchActionButtonVisibility = false
+    }
+    assert.strictEqual(
+      deleteBatchActionButtonVisibility,
+      false,
+      `Expected delete action button not to be visible but was `
+    )
+  }
+)
+
+When('the user selects the file/folder {string} using the webUI', function (fileName) {
+  return client.page.FilesPageElement.filesList().toggleFileOrFolderCheckbox('enable', fileName)
+})
+
+Then(
+  'the following batch action buttons should be visible on the shared with me page',
+  async function (dataTable) {
+    const batchButtons = dataTable.hashes()
+    for (const batchButton of batchButtons) {
+      const isBatchButtonVisible = await sharedWithMePage.isBatchButtonVisible(batchButton.buttons)
+      assert.strictEqual(
+        isBatchButtonVisible,
+        true,
+        `Expected to board list to be ${batchButton.buttons} but was not`
+      )
+    }
+  }
+)
+
+Then(
+  'the following batch action buttons should be visible on the trashbin page',
+  async function (dataTable) {
+    const batchButtons = dataTable.hashes()
+    for (const batchButton of batchButtons) {
+      const isTrashbinBatchButtonVisible = await trashbinPage.isActionButtonVisible(
+        batchButton.buttons
+      )
+      assert.ok(
+        isTrashbinBatchButtonVisible,
+        `Expected "${batchButton.buttons}" action button to be visible but was not`
+      )
+    }
+  }
+)
+
+Then('the batch action button should not be visible on the favorites page', async function () {
+  let batchActionButtonPresentStatus = await favoritesPage.getBatchActionButtonPresentStatus()
+  if (batchActionButtonPresentStatus === -1) {
```

Comment:
please parse output of `getBatchActionButtonPresentStatus` fn to be a boolean value.

Reply:
There is no built in function to parse -1 to boolean value as false